src/emailalert/email_function.py

```diff
@@ -25,6 +30,8 @@ def lambda_handler(event, context):
 
     CHARSET = "UTF-8"
     client = boto3.client('ses',region_name=AWS_REGION)
+    logger.info('## HTML BODY AND CHARSET')
```

Comment:
this is a constant. does not need to be logged.
instead please log the final error message that is being sent (captured in `result` object)